tsdb/engine/tsm1/compact.go

```diff
@@ -301,7 +303,7 @@ func (c *Compactor) Compact(tsmFiles []string) ([]string, error) {
 		return nil, nil
 	}
 
-	tsm, err := NewTSMKeyIterator(trs...)
+	tsm, err := NewTSMKeyIterator(tsdb.DefaultMaxPointsPerBlock, trs...)
```

Comment:
Yes.  That is the size of blocks that it will emit.  It doesn't matter what the block sizes are within the files it that it reads.